src/it/scala/io/iohk/ethereum/db/LevelDBDataSourceIntegrationSuite.scala

```diff
@@ -5,11 +5,12 @@ import org.scalatest.FlatSpec
 
 class LevelDBDataSourceIntegrationSuite extends FlatSpec with DataSourceIntegrationTestBehavior {
 
-  private def createDataSource(path: String) = LevelDBDataSource(path, new LevelDbConfig {
+  private def createDataSource(path: String) = LevelDBDataSource(new LevelDbConfig {
     override val cacheSize: Int = 0
     override val verifyChecksums: Boolean = true
     override val paranoidChecks: Boolean = true
     override val createIfMissing: Boolean = true
+    override val path: String = "/tmp/test/leveldb/"
```

Comment:
Shouldn't this `val path` use the `path` parameter? Because if not I think we might not we using the randomly created directories from the tests.